objectivec/GPBBootstrap.h

```diff
@@ -95,7 +95,11 @@
  * Attribute used for Objective-C proto interface deprecations.
  **/
 #ifndef GPB_DEPRECATED
-#define GPB_DEPRECATED __attribute__((deprecated))
```

Comment:
Won't removing this be source breaking if someone had generated something with the pervious change that switched to it?  i.e. - I think you need to keep `GPB_DEPRECATED` and add a new `GPB_DEPRECATED_MSG(...)` and keep the old just for safety sake.

Reply:
Good catch (and sorry that I didn't think to support the message in my initial addition of `GPB_DEPRECATED`). Created a separate `GPB_DEPRECATED_MSG`.

Reply:
Done.